src/kibana/components/agg_types/controls/precision.html

```diff
@@ -16,4 +16,16 @@
       </div>
     </div>
   </div>
-</div>
\ No newline at end of file
+</div>
+
+<div class="vis-option-item">
+  <label>
+    <input type="checkbox"
+    name="autoPrecision"
+    value="{{autoPrecision}}"
```

Comment:
Yes, ng-model takes care of the value and ng-checked for both of these inputs. Did not know that. Thanks!